### PR TITLE
Increase Heap size to fix test timeouts

### PR DESCRIPTION
**Pull request by bhhari**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by bhhari** [comment]
fixes #14092 14092

File: presto-tests/pom.xml [hunk 1]
```diff
@@ -216,6 +216,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <argLine>-Xms4g -Xmx4g</argLine>
```
Comment (ajaygeorge) [inline presto-tests/pom.xml]: I think this needs to be included in the correct profile. I see two profiles in the same file. `ci` and `ci-only` . Do we know which is the correct profile.?

Comment (bhhari) [reply presto-tests/pom.xml]: This should apply to both right if we specify here right?

Comment (ajaygeorge) [reply presto-tests/pom.xml]: 👍  LGTM

**Pull request merged**